### ISSUE TITLE
Brink [BLES01304] | VM: Access violation reading location 0x0 (unmapped memory)

### ISSUE THREAD

**Issue by LoStraniero91** [RPCS3/rpcs3#11945]
**RPCS3 version:** 0.0.22-13562
**System specs:** i9-9900k/32 GB/RTX 2080 Ti/Windows 10

Brink has never worked so far on RPCS3 and it seems that nobody has ever reported this issue here on Github. Tried a combination of PPU/SPU settings and also Debug Console Mode. It always hangs in the first loading screen.

**Log**
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8635899/RPCS3.log.gz)

**Comment by Megamouse** [comment]
compatibility reports belong on the forums

**Comment by Megamouse** [comment]
I guess I can leave it open

**Comment by kd-11** [comment]
Odd that it doesn't run, we usually don't have compatibility problems with idtech titles.

**Comment by Xcedf** [comment]
Ingame with LLE libaudio
![30](https://user-images.githubusercontent.com/7060586/167120930-294873b3-d038-4fe2-971c-3fea3c1dddc6.jpg)

**Comment by Darkhost1999** [comment]
From what I understand LLE Libaudio is not officially supported by RPCS3 at this told so no audio makes sense as nothing is hooked up to anything to make noise.
True an audio label and an HLE label would work here as HLE would be the problem if LLE works. Thanks for keeping us updated. Just wanted to let you know why audio isn't outputting anything.

**Comment by digant73** [comment]
is that still an issue?

**Comment by TigiHof** [comment]
> is that still an issue?

Yes, still broken in 0.0.38-18263. Brink goes ingame only when checking libaudio.sprx (otherwise  the game crashes), but sound is not working.
